### PR TITLE
Add license.txt

### LINKED ISSUE #114: no LICENSE.txt for this repository
This repository doesn't seem to have a [license file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository#:~:text=You%20can%20include%20an%20open,for%20other%20people%20to%20contribute.). This makes it difficult and unclear for others to freely use, change, or distribute the project.

Would it be possible to add a LICENSE.TXT. Something permissive like MIT License would be great!

### PR DESCRIPTION
**Pull request by chrbertsch**
resolves #114

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+The FMUs uploaded to this repository are provided to for testing purposes within the FMI Cross Check only. For this purpose the FMUs may be simulated by in importing tools.
```
Comment (klausschuch) [inline LICENSE.txt:1]:
```suggestion
The FMUs uploaded to this repository are provided for testing purposes within the FMI Cross Check only. For this purpose the FMUs may be simulated by importing tools.
```

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+Typically, all further rights are reseverd by the tool vendors who upload the FMUs.
+More information can be found in the subfolders under https://github.com/modelica/fmi-cross-check/tree/master/fmus and/or the docuementation folder of the FMUs.
```
Comment (klausschuch) [inline LICENSE.txt:4]:
```suggestion
More information can be found in the subfolders under https://github.com/modelica/fmi-cross-check/tree/master/fmus and/or the documentation folder of the FMUs.
```

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+It is in the responsibility of the providers of the FMUs to comply with license conditions of third parties, e.g., w.r.t open source licenses of software contained in the FMUs or in used in the creation of the FMUS, e.g., by shipping the corresponding license texts within the documentation folder FMU.
```
Comment (klausschuch) [inline LICENSE.txt:6]:
```suggestion
It is in the responsibility of the providers of the FMUs to comply with license conditions of third parties, e.g., w.r.t. open source licenses of software contained in the FMUs or used in the creation of the FMUs, e.g., by shipping the corresponding license texts within the documentation folder of the FMU.
```

**Review by klausschuch — CHANGES_REQUESTED** [review]

**Review by andreas-junghanns — APPROVED** [review]
@klausschuch : ok now? Can we merge?

**Pull request merged**